teacher.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'person'
 class Teacher < Person
-  def initialize(specialization)
-    super(id, age, name, parent_permission: true)
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
     @specialization = specialization
   end
```

Comment:
- [ ]  Good job to create person, but seemed like there one issue happening, that when I try to add specialization to a teacher. Here is the issue that you need to fix for your app to work properly. 
![image](https://github.com/zunairkhan811/OOP_School_Library/assets/43877689/80ac552d-395f-409b-a4c9-e8460e897d19)

Reply:
Hello @zunairkhan811 , My apology that I mistakenly approved your project in the dashboard, but you need to work on the requested changes for you to proceed and merge. Thank you for listening to me .

Reply:
Thanks! it has been fixed now